src/ledgerhelpers/gui.py

```diff
@@ -40,7 +40,7 @@ def add_css(css):
     global _css_adjusted
     if css not in _css_adjusted:
         style_provider = Gtk.CssProvider()
-        style_provider.load_from_data(css)
```

Comment:
This one needs a check where if the `css` variable is actually a `bytes` object.  Else you just use `css` as-is.